update existed tutorial for signer (#351)
paritytech/wiki: Parity-Signer-Mobile-App-Fether-tutorial.md

## 2. Setup or recover an account
When launching the app for the first time, no account has been set up yet. At this stage, you will either want to create an identity (root account) directly from your mobile device or recover an identity (root account) previously created with Parity Signer or [Parity UI / Wallet](https://wiki.parity.io/Parity-Wallet) (Desktop)
 
### Create an account
 
Tap on the `Create` button after you first entered into the app, and then you can give the identity any name you like.

In the next step, your recovery phrase will be presented to you. Think of it as a master key. If you lose it, you lose your money.
**Write this recovery phrase down and store it in a safe place**.
If your phone gets stolen/broken/forgotten this will be the only way to [recover your account](#recover-an-account-with-your-recovery-phrase).

You will then be asked to choose a pin code. This pin will be needed later on to unlock your account to manage the identity or sign a transaction.

The next screen you will allow you to select a network for generating an account.

![create account](images/Parity-Signer-apps-0.png)

### Recover an account with your recovery phrase

> If you already have an account created with either Parity Signer or any other waller, you can recover it by doing so:

Reviewer: ```suggestion
If you already have an account created with either Parity Signer or any other wallet, you can recover it by doing so:
```

- Tap on the top right side user icon, and choose ` + Add Identity`.
- Input the new identity name and tap the `Recover Identity` button.
- Type in the recovery phrase, word suggestion helps you prevent any typo. The field will turn red if the recovery phrase is not a bip39.
- Tap `Recover Identity`.
- Select a PIN number and confirm it by typing it again.

> - Identity generated, now you can select the network to create the first account.

Reviewer: ```suggestion
- Once your Identity is generated, you can select a network to create your first account.
```
